crates/protocol/src/packets.rs

```diff
@@ -0,0 +1,11 @@
+//! Hand-written packet types
+
+pub mod cb;
+pub mod sb;
```

Comment:
We should not import them by name. I'd rather put a doc comment indicating what these mean

Reply:
> Since we can import them by name

I meant individual packets from these packets. Still `cb` and `sb` sound to hard to understand for me without prior knowlwdge on commonly used names.

Reply:
What should we do in `net_thread.rs`, where both clientbound and serverbound packets will be needed, often with clashing names?

Reply:
We could ad-hoc re-export them as `use ...::clientbound as cb` while keeping the public API more self-defining.

Reply:
Done; renamed to `c2s` and `s2c`